### PR TITLE
Add SNS event handler support

### PR DESCRIPTION
**Pull request by mnapoli**
This is a continuation of #521 to add support for SNS event handlers.

### CODE REVIEW

File: src/Event/Sns/MessageAttribute.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * @see https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/SNSEvent.java
+ */
+class MessageAttribute
+{
+    /** @var array */
+    private $attribute;
+
+    public function __construct(array $attribute)
+    {
+        $this->attribute = $attribute;
```
Comment (AlessandroMinoccheri) [inline src/Event/Sns/MessageAttribute.php]: May it be necessary to check that there is an Index Type and Value?

Comment (mnapoli) [reply src/Event/Sns/MessageAttribute.php]: I'm not sure, at this point the shape of the event should have been validated earlier. I don't validate all keys in all value objects.

**Comment by mnapoli** [comment]
Tests are failing because of a difference in date format in PHP 7.2 it seems 🤔

File: src/Event/Sns/MessageAttribute.php (hunk 1 above)
Comment (AlessandroMinoccheri) [reply src/Event/Sns/MessageAttribute.php]: ok, usually I validate all keys to ensure the correct use of the value object. But it's not a block for this PR I think

**Pull request merged**